### PR TITLE
Use six for pulling in unquote.

### PR DESCRIPTION
**Pull request by dsully**

### CODE REVIEW

**Comment by brennie** [comment]
Hey, thanks for this. However, we do not accept pull requests. Can you post this over on https://reviews.reviewboard.org? Thanks!

**Comment by dsully** [comment]
You're not serious, are you?

**Comment by dsully** [comment]
I assume you've had this conversation often, and would explain why there are no PRs or issues here.

Posted on reviews.reviewboard.org..

**Comment by chipx86** [comment]
Hi,

While GitHub Pull Requests are a popular GitHub feature for reviewing code, we (hopefully understandably) use Review Board to review changes going into Review Board. This allows us to ensure that all code is properly seen by all developers/contributors following our Review Board server, keeps all contributions in one place, and helps us to dogfood our product. If we were to allow patches to come in either on Review Board *or* GitHub Pull Requests, we'd be making things more confusing and difficult for everybody involved with the project, and sending mixed messages about what code review product is "correct" to use.

It also helps us ensure that our changes are [formatted according to our standards](https://www.reviewboard.org/docs/codebase/dev/writing-good-descriptions/) (having a separate summary, change description, and information on the testing that was done), and to ensure it's landed in a consistent way (stamped with the review request URL). This helps us greatly when going through changes for the purpose of writing release notes, bisecting, etc., and keeps the repository clean, tidy, and informative.

Eventually it'll be possible to create a pull request and have that automatically pulled into Review Board, which will help in these situations, but we'd still be performing the reviews there.

**Comment by chipx86** [comment]
As for issue tracking, we use [our own bug tracker](https://hellosplat.com/s/beanbag/) (which is still heavily in development) for all bug reports. This is partly because we had a whole database of bug reports that existed before GitHub was created that we didn't want to throw out, and importing them into GitHub proved to be a bad experience (too many sacrifices had to be made about the content and the abilities we wanted to keep). We also have some big plans for our bug tracker, Splat.

Basically, we use GitHub solely for code storage. If we had the ability to turn off the other features, we'd do that.
